api/lib/models/checkout-items/checkout-items.ts

```diff
@@ -12,19 +15,21 @@ interface ICheckoutItemsApiModel {
 }
 
 export class CheckoutItems extends BaseObject {
-
-public name: string;
-public quantity: number
-
+  
   public get schema() {
-    return null;
+    return checkoutItemsSchema;
   }
   public get id() {
-    return null
+    return undefined;
```

Comment:
As per our discussion, if at some points we need an ID field for checking the availability, maybe this should actually be a defined property that doesn't get set. Correct me if I'm wrong though